.github/workflows/test-server-all.yml

```diff
@@ -185,9 +181,8 @@ jobs:
         run: make test t=./manage_container m=run p=medium
         working-directory: ./scripts
 
-  medium_test_libclient:
+  medium_test_libclient_test_correl:
```

Comment:
Since everything is the same except for test name, a matrix strategy may be used.

doc: https://docs.github.com/ja/actions/using-jobs/using-a-matrix-for-your-jobs
example: https://github.com/acompany-develop/QuickMPC/blob/da5b6eadb9335d95f9bea53b7687a271316cdf83/.github/workflows/test-client-libclient-py.yml#L45-L53

Reply:
Do you have the following implementations in mind?
```
medium_test_libclient:
    strategy:
      matrix:
        test_name: [correl, get_elapsed_time, ...]
    steps:
      ...
      - run: ... pytest src/tests/test_${{ JOB_NAME }} ...
        env:
          JOB_NAME: ${{ matrix.test_name }}
```

Reply:
@mdonaka 
libc : [debb744](https://github.com/acompany-develop/QuickMPC/pull/197/commits/debb744d518956be72782ff5f7546d85a9006e73)
container_{restart,down} : [245868a](https://github.com/acompany-develop/QuickMPC/pull/197/commits/245868a3f9a756da6e11a02c5a419ef321973c10)